app/controllers/user_controller.rb

```diff
@@ -2,7 +2,8 @@
 class UserController < ApplicationController
   def index
     if user_signed_in?
-      render :show, params: { username: current_user.username }
+      @user_profile = User.find_by( username: current_user.username )
```

Comment:
Can the `find_by` be avoided?

```diff
-@user_profile = User.find_by( username: current_user.username )
+@user_profile = current_user
```

Reply:
Oh wow. How embarrassing is that! Coding when tired :rofl: